packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -891,3 +891,135 @@ import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-i
   </DescriptionListGroup>
 </DescriptionList>;
 ```
+
+### Column fill
```

Comment:
Adding the isFillColumns prop to this example does not change the default behavior, right? Is this example necessary?

Reply:
In the same vein, if we are to have an example, does 1 demonstrating this prop's behavior not suffice (vs the 3 I'm seeing here)?

Reply:
@jeffpuzzo we typical add examples to match the core implementation.  To reduce the duplicate example we could add check box to already existing `Default` and `Horizontal` example to toggle the `pf-m-fill-columns` modifier. We could also add a `Horizontal multi-column` example with a checkbox toggle since one does not exist.  Or we could add could use one of the other examples that happens to have horizontal multi columns.

cc @mcoker  thoughts on removing the example that really does not change the default behaviour.

Reply:
@tlabaj agree, it's confusing because it doesn't demonstrate anything. We could have a regular and horizontal example, though I'm fine with just having a regular description list with multiple columns.